app/assets/javascripts/uswds.js

```diff
@@ -0,0 +1,14 @@
+var button = document.getElementsByClassName("usa-banner__button")[0];
+var banner_content = document.getElementsByClassName("usa-banner__content")[0];
+
+button.addEventListener("click", function(){
+    var is_hidden = banner_content.getAttribute("hidden");
```

Comment:
@yogesh27 , our existing JS files use two spaces for indentation in JS files. Unless you feel strongly that we should use 4, I think we should make 2 the default (let's discuss general JS linting w/ the team at scrum). Our Ruby files all use 2 spaces, which seems to be turning into the standard for JS, too: https://www.w3schools.com/js/js_conventions.asp

Reply:
We can use 2 spaces. I will update the editor settings.